app/templates/gentelella/guest/home.html

```diff
@@ -18,8 +18,11 @@
         }
 
         .event-holder img {
-            width: 100%;
-            height: 165px;
```

Comment:
This will ensure image is not stretched and at the same time will make sure there is no ugly white space in the side.